ngsi_adapter/src/test/acceptance/features/component/send_data/send_data_api_resource.feature

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+Feature: Sending probe data
+  As a monitoring element (probe) user
+  I want to be able to transform monitoring data from probes to NGSI context attributes
+  so that I can constantly check the status and performance of the cloud infrastructure using NGSI Context Broker.
+
+
+  @happy_path
+  Scenario: Probe sends valid raw data using an existing and well-formed parser
```

Comment:
I'd say "Valid probe data is sent to CB using...". The probe itself sends data to Nagios; data is intercepted by module NGSIEventBroker, which sends it to NSGIAdapter.

(This comment applies to all scenarios below).